src/pygpsclient/spartn_mqtt_frame.py

```diff
@@ -257,6 +264,13 @@ def _do_layout(self):
         self._btn_connect.grid(column=0, row=9, padx=3, pady=2, sticky=W)
         self._btn_disconnect.grid(column=2, row=9, padx=3, pady=2, sticky=W)
 
+    def _attach_events(self):

```

Comment:
Delete this function if it's not being used

Reply:
done